VideoTransformers/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Windows;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("VideoTransformers")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("VideoTransformers")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
```

Comment:
NIT: 2023,  we need to make a way to share all these properties across all samples so we can update them easily. Don't change it now since it's better to do it the correct way.

Reply:
Yes I agree